src/networks.json

```diff
@@ -2222,5 +2222,22 @@
     },
     "start": 498623,
     "logo": "ipfs://QmSwCnEeLoyMApQd1YfhmGvLt97bH5mLPoabjKqUSuQHHF"
+  },
+  "12345": {
+    "key": "12345",
```

Comment:
Is there some documentation for this network that i can refer to? because i can't find it on https://chainlist.org/?search=12345&testnets=true